src/dbnode/storage/shard_insert_queue.go

```diff
@@ -33,6 +33,7 @@ import (
 	"github.com/m3db/m3/src/x/checked"
 	"github.com/m3db/m3/src/x/ident"
 	xtime "github.com/m3db/m3/src/x/time"
+	"go.uber.org/zap"
```

Comment:
nit: Move to third group of imports, also move `"github.com/m3db/m3/src/dbnode/namespace"` to second group of imports.